src/pdf/DriverOverview/DriverOverviewPdf.tsx

```diff
@@ -248,27 +274,38 @@ const DriverOverviewCard: React.FC<DriverOverviewCardProps> = ({ data }) => {
 
     return (
         <Document>
-            <Page size="A4" style={[styles.container, styles.flexColumn]}>
-                <View style={[styles.infoAndLogoContainer, styles.flexRow]}>
-                    <View style={styles.informationContainer}>
-                        <Text style={styles.h1text}>Driver Overview</Text>
-                        <Text style={styles.h2text}>
+            <Page size="A4" orientation="landscape" style={[styles.container, styles.flexColumn]}>
+                <View style={styles.DriverOverviewBoard}>
+                    <View style={{ flexDirection: "column", fontFamily: "Helvetica-Bold" }}>
```

Comment:
there are quite a few views where you add the fontFamily: "Helvetica-Bold" styling. If this is used everywhere on the PDF, is it possible to move this so it is the default font for the page, to reduce the number of times you need to define the font?

Reply:
There's still a place where I used helvetica (not bold), so maybe not setting the bold one as default is better?